doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,157 @@
+# Submit a Distributed Training Job
+
+If a user wants to start up a local train, he will start up a PaddlePaddle product Docker container firstly, and then
+execute `python train.py` in the Docker container.The details about PaddlePaddle Docker image is [here](../../../paddle/scripts/docker/README.md)
+
+If a user wants to start up a distributed training job, he will submit the distributed training job in python code, or use a command line tool.
+
+The relation of PaddlePaddle, kubernetes and docker:
+
+
+# Runtime Environment On kubernetes
+
+For a distributed training job, there is two docker image called `runtime docker image` and `base docker image`, the `runtime docker image` is actually running in kubernetes.
+
+- Base Docker Image
+
+  Usually, the `base docker image` is PaddlePaddle product docker image including paddle binary files and trainer startup script file. And of course, users can specify any image name hosted on any docker registry which users have the right access.
+
+- Runtime Docker Image
+
+  Package the trainer package which user upload and some python dependencies into a `runtime docker image` base on `base docker image`, this is done automatically by Job Server.
+
+- Python Dependencies
+
+  Users will provide a `requirments.txt` file in trainer packages, to list python dependencies packages, such as:
+  ```txt
+  pillow
+  protobuf==3.1.0
+  ```
+  some other details about `requirements` is [here](https://pip.readthedocs.io/en/1.1/requirements.html).
+
+  Here is an example project:
+  ```bash
+    paddle_example
+      |-quick_start
+        |-trainer.py
+        |-dataset.py
+        |-requirments.txt
+  ```
+  Execute the command: `paddle train --trainer-package=./paddle_eample/quick_start ...`, PaddlePaddle client will upload the trainer package(quick_start)and setup parameters to [Job Server](#job-server)
+
+## Submit a Distributed Training Job In Python Code
```

Comment:
Submit a Distributed Training Job In Python Code -> Submit Distributed Training Job With Python Code